crypto/bio/bio_dump.c

```diff
@@ -141,9 +141,10 @@ int BIO_hex_string(BIO *out, int indent, int width, const void *data,
 
         BIO_printf(out, "%02X:", d[i]);
 
-        j = (j + 1) % width;
-        if (!j)
+        if (++j >= width) {
+            j = 0;
```

Comment:
This is a needless optimisation - the execution speed here will be dominated by the calls to `printf()`.

It's worthwhile making this change if it makes the intent clearer, which I'm on the fence about - what do other people think?

Reply:
I'm not sure it does particularly. It doesn't make it any worse, but I'm not sure it makes it better either.

Reply:
It at least corrects a coding style error as `if (!j)` should be written as `if (j == 0)`

Reply:
Yeah. Ok.

Reply:
That same style error is at line 139